scale/source/urls.py

```diff
@@ -9,5 +9,6 @@
     url(r'^sources/updates/$', views.SourceUpdatesView.as_view(), name='source_updates_view'),
     url(r'^sources/(?P<source_id>\d+)/$', views.SourceDetailsView.as_view(), name='source_details_view'),
     url(r'^sources/(?P<file_name>[\w.]{0,250})/$', views.SourceDetailsView.as_view(), name='source_details_view'),
+    url(r'^sources/(?P<source_id>\d+)/jobs/$', views.SourceJobsView.as_view(), name='source_jobs_view'),
```

Comment:
Since the sources details view supports an ID or file name (requested by users), it would be good if any of the sub-resources also supported file name for consistency.